titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/v1/V1SpecPodFactory.java

```diff
@@ -460,7 +458,6 @@ V1ResourceRequirements buildV1ResourceRequirements(ContainerResources containerR
         });
 
         appendS3WriterRole(annotations, job, task);
-        annotations.putAll(createEbsPodAnnotations(job, task));
```

Comment:
I believe these annotations are still going to used by our kube exclusive resource scheduler plugin (https://github.com/Netflix/titus-kube-common/blob/master/pod/annotations.go#L104). Is there something else that we are expecting will enforce exclusive access to the V1Volume/PVC?

Reply:
Oh. Hmm.
I suppose it would be nice if that plugin respected V1 volumes and didn't have to rely on a custom annotation?

If you point me to the code, maybe I can add in that capability first?

Reply:
Here is the plugin code (https://stash.corp.netflix.com/projects/TN/repos/titus-kube-scheduler/browse/plugins/exclusive/ebsvolid/ebsvolid_plugin.go) and CC @joshi-keyur who originally wrote it.

Reply:
Hmm. Looks very annotation-specific.

I'll discard for now